ui/page/main_page.go

```diff
@@ -328,6 +332,12 @@ func (mp *MainPage) Handle() {
 			mp.changeFragment(pg)
 		}
 	}
+
+	mp.isBalanceHidden = mp.WL.MultiWallet.ReadBoolConfigValueForKey("hide Balance", false)
+	if mp.hideBalance.Button.Clicked() {
+		mp.isBalanceHidden = !mp.isBalanceHidden
+		mp.WL.MultiWallet.SetBoolConfigValueForKey("hide Balance", mp.isBalanceHidden)
+	}
```

Comment:
```suggestion
	mp.isBalanceHidden = mp.WL.MultiWallet.ReadBoolConfigValueForKey(HideBalanceConfigKey, false)
	if mp.hideBalance.Button.Clicked() {
		mp.isBalanceHidden = !mp.isBalanceHidden
		mp.WL.MultiWallet.SetBoolConfigValueForKey(HideBalanceConfigKey, mp.isBalanceHidden)
	}
```

make the config key to be constants.

also config keys should be separated by hyphen e.g `HideBalanceConfigKey = "hide_balance"`